src/components/policy/policy_regular/include/policy/policy_helper.h

```diff
@@ -83,22 +83,11 @@ bool operator!=(const policy_table::ApplicationParams& first,
 struct CheckAppPolicy {
   CheckAppPolicy(PolicyManagerImpl* pm,
                  const std::shared_ptr<policy_table::Table> update,
-                 const std::shared_ptr<policy_table::Table> snapshot);
+                 const std::shared_ptr<policy_table::Table> snapshot,
+                 CheckAppPolicyResults& out_results);
   bool operator()(const AppPoliciesValueType& app_policy);
 
  private:
-  enum PermissionsCheckResult {
```

Comment:
Should this have been removed?

Reply:
Yes, it was a duplicate of enum in `policy_types.h`